HW_6/models.py

```diff
@@ -0,0 +1,48 @@
+''' Таблица пользователей должна содержать следующие поля: id (PRIMARY KEY), имя, фамилия, адрес электронной почты и пароль.
+• Таблица заказов должна содержать следующие поля: id (PRIMARY KEY), id пользователя (FOREIGN KEY), id товара (FOREIGN KEY), дата заказа и статус заказа.
+• Таблица товаров должна содержать следующие поля: id (PRIMARY KEY), название, описание и цена.'''
+
+from pydantic import BaseModel, Field
+
+
+class UserIn(BaseModel):
+    first_name: str = Field(max_length=40)
+    last_name: str = Field(max_length=40)
+    email: str = Field(max_length=80)
+    password: str = Field(min_length=20)
+
+
+class User(BaseModel):
+    id: int
+    first_name: str = Field(max_length=40)
+    last_name: str = Field(max_length=40)
+    email: str = Field(max_length=80)
+    password: str = Field(min_length=20)
```

Comment:
Модели очень похожи, а значит можно создать одну из них, наследуясь от другой.
```suggestion
class User(UserIn):
    id: int
```